static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -653,6 +653,7 @@
     }
 }
 
+// This is now injected via JS in static/src/javascripts/projects/commercial/modules/messenger/background.js
```

Comment:
It's only injected for DCR, right? Might be worth specifying that here so that this isn't removed in error for Frontend

Reply:
Fair point.